controlplane/kubeadm/config/webhook/service.yaml

```diff
@@ -0,0 +1,12 @@
+
+apiVersion: v1
+kind: Service
+metadata:
+  name: webhook-service
+  namespace: system
+spec:
+  ports:
+    - port: 443
+      targetPort: 9443
+  selector:
+    control-plane: controller-manager
```

Comment:
this will not be replaced by kustomize, must manually set this to the value kustomize will create...not nice but it's what we do for capi and I suppose the best workaround.